src/target/riscv/riscv.h

```diff
@@ -248,6 +248,10 @@ typedef struct {
 	 * from range 0xc000 ... 0xffff. */
 	struct list_head expose_custom;
 
+	/* The list of registers to mark as "hidden". Hidden registers are available
+	 * but do not appear in gdb targets description or reg command output. */
+	struct list_head hidden_csr;
```

Comment:
```suggestion
	struct list_head hide_csr;
```
To keep naming consistent with `expose_csr`.

Reply:
addressed